### PR TITLE
:tada: init gradle

### PR DESCRIPTION
**Pull request by ShuffleZZZ**

### CODE REVIEW

File: src/main/java/Main.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import com.elbekD.bot.Bot
```
Comment (nothingelsematters) [inline src/main/java/Main.kt:1]: I suggest, that we name folders as `src/main/kotlin`. 

That's the way it's handled in the following projects:

+ [ktlint](https://github.com/pinterest/ktlint/tree/master/ktlint/src/main)

+ [codecov example](https://github.com/codecov/example-kotlin)

**Comment by DubKoldun** [comment]
we have a .bat file, for what?

File: README.md [hunk 2]
```diff
@@ -2,4 +2,8 @@
 
 [![Build Status](https://travis-ci.com/pool-party/fucker-bot.svg?branch=master)](https://travis-ci.com/pool-party/fucker-bot) [![codecov](https://codecov.io/gh/pool-party/fucker-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/pool-party/fucker-bot) [![CodeFactor](https://www.codefactor.io/repository/github/pool-party/fucker-bot/badge)](https://www.codefactor.io/repository/github/pool-party/fucker-bot)
 
-telegram bot notifier
\ No newline at end of file
+Run ```gradlew clean build``` first time to setup required version of gradle and dependencies.
```
Comment (DubKoldun) [inline README.md:5]:
```suggestion
Run ```./gradlew clean build``` first time to setup required version of gradle and dependencies.
```

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1 @@
+rootProject.name = 'fucker-bot'
```
Comment (DubKoldun) [inline settings.gradle:1]:
```suggestion
rootProject.name = 'pull-party-bot'
```

Comment (nothingelsematters) [reply settings.gradle:1]: You can change `README.md` as well

**Comment by nothingelsematters** [comment]
@DubKoldun I suggest the following message for your last commit: `:see_no_evil: update gitignore`

File: README.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
-# Fucker bot
+# Pull party bot
 
 [![Build Status](https://travis-ci.com/pool-party/fucker-bot.svg?branch=master)](https://travis-ci.com/pool-party/fucker-bot) [![codecov](https://codecov.io/gh/pool-party/fucker-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/pool-party/fucker-bot) [![CodeFactor](https://www.codefactor.io/repository/github/pool-party/fucker-bot/badge)](https://www.codefactor.io/repository/github/pool-party/fucker-bot)
 
-telegram bot notifier
\ No newline at end of file
+For the first time to setup required version of gradle and dependencies run:
+```./gradlew clean build``` for UNIX systems
+```gradlew clean build```   for windows
```
Comment (nothingelsematters) [inline README.md:7]: `./gradlew.bat` is for windows, isn't it?

Comment (ShuffleZZZ) [reply README.md:7]: > `./gradlew.bat` is for windows, isn't it?

Nope

**Pull request merged**